contents/javascript/javascript.md

```diff
@@ -0,0 +1,272 @@
+# Writing Testable Javascript
+
+JavaScript is a powerful language. However, its flexibility leads to multiple ways for people to go about doing the same thing. The end result is that multiple collaborators working on a single project can produce different code that does the same thing.
+
+That is why there is a need to follow a standard way of writing JavaScript - it allows for more maintainable cleaner and more beautiful code.
+
+Good JavaScript code should be testable, composable and reusable.
```

Comment:
In any case, it's not addressed in the rest of the document. Only testable and reusable are addressed.